ui/inspectors/share.reel/share.html

```diff
@@ -70,8 +70,8 @@
                 "element": {"#": "overview"}
             },
             "bindings": {
-                "object": {"<-": "@owner.object"},
-                "targetType": {"<->": "@owner.targetType"}
+                "title": {"<-": "@owner.object.type.toUpperCase()"},
```

Comment:
The way the overview displays the type (uppercased, here) is not inspector's concern, it should be moved to the overview component.